packages/ember-views/lib/views/bound_component_view.js

```diff
@@ -0,0 +1,50 @@
+/**
+@module ember
+@submodule ember-views
+*/
+
+import { _Metamorph } from "ember-views/views/metamorph_view";
+import { read, chain, subscribe, unsubscribe } from "ember-metal/streams/utils";
+import { readComponentFactory } from "ember-views/streams/utils";
+import mergeViewBindings from "ember-htmlbars/system/merge-view-bindings";
+import EmberError from "ember-metal/error";
+
+export default Ember.ContainerView.extend(_Metamorph, {
```

Comment:
Updated the [JSBin](http://emberjs.jsbin.com/rwjblue/188/edit) to share the same morph (thanks to @mmun for pointing out the extra morhps being created in the first one).